code/game/objects/items/melee/energy.dm

```diff
@@ -119,7 +119,6 @@
 		heat = initial(heat)
 		STOP_PROCESSING(SSobj, src)
 
-	tool_behaviour = (active ? TOOL_SAW : NONE) //Lets energy weapons cut trees. Also lets them do bonecutting surgery, which is kinda metal!
```

Comment:
:(

Reply:
Should be a var tbh

Reply:
laser scalpel does this same thing because it's heat-related. let me melt through bones!